.github/workflows/comment.yml

```diff
@@ -0,0 +1,15 @@
+name: Comment-triggered workflow
+on: 
+    issue_comment:
+        types: [created, edited]
+jobs:
+  pr_commented:
+    name: PR comment
+    if: ${{ github.event.issue.pull_request }}
```

Comment:
might be `if: github.event.issue.pull_request && contains(github.event.comment.body, '/<your-desired-command>')`?

Reply:
Based on Chris Cook's article

Reply:
I've tried that too, but it did not work. At this point I just want it to trigger for any comment :D